traveler-swift-core/TravelerKit/Models/ParkingItemDetails.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ParkingItemDetails.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-10-04.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// The detailed information of a parking `ProductItem`
+public struct ParkingItemDetails: CatalogItemDetails, Product, Decodable {
+    /// Description
+    public let description: String?
+    /// An array of URLs of images
+    public let imageUrls: [URL]
+    /// Attributes
+    public let information: [Attribute]?
+    /// Vendor's contact information
+    public let contact: ContactInfo?
+    /// An array of locations
+    public let locations: [Location]
+    /// Product supplier
+    public let supplier: Supplier
+    /// Disclaimer
+    public let disclaimer: String?
+    /// Terms and conditions
+    public let termsAndConditions: String?
+    /// Identifier
+    public let id: String
+    /// Price
+    public let price: Price
+    /// Type
+    public let type: ProductType
+    /// Title
+    public let title: String
+    /// Amount to be paid online
+    public let paybleOnline: Price
+    /// Amount to be paid on site
+    public let payableOnSite: Price
```

Comment:
these two are confusing...`priceToPay` or `priceToPayOnsite` or `pricePayableOnSite`...its not clear if this is boolean, price, etc.

Reply:
Also how does `price` itself fit into this...There are three different price attributes...this already confused me.

Reply:
price is the starting price, payableOnline (now called priceToPayOnline) is the amount you would pay online, payableOnSite(now called priceToPayOnsite) is the amount you would pay on site. There's 3 different kids of prices for this model.